src/gwt/src/org/rstudio/studio/client/workbench/views/output/find/FindInFilesDialog.java

```diff
@@ -56,14 +56,14 @@ public static native State createState(String query,
                                              String path,
                                              boolean regex,
                                              boolean caseSensitive,
-                                             boolean excludeIgnore,
+                                             boolean gitIgnore,
```

Comment:
Does this bool mean you should or shouldn't read the .gitIgnore file?

Reply:
If it's true, then we should read the gitIgnore. Right now this variable has no functionality and is really just a placeholder for a future PR.